src/server/db/story-dbConfiguration.js

```diff
@@ -141,7 +151,7 @@ sjrk.storyTelling.server.storiesDb.storyTagsFunction = function (doc) {
  * @param {Object} doc - a document to evaluate in the view
  */
 sjrk.storyTelling.server.storiesDb.storiesByIdFunction = function (doc) {
-    if (doc.value.published) {
+    if (doc.type === "story" && doc.value.published) {
```

Comment:
The condition of checking `doc.value.published` shows a more meaning name for this view might be "publishedStoriesById`.

Reply:
Makes sense. I've renamed it.